recipe/meta.yaml

```diff
@@ -79,17 +79,14 @@ outputs:
         - attrs
         - click >=7.0, <=8.0.4
         - colorama
-        - dataclasses  # [py<37]
         - filelock
         - frozenlist
-        # We override the 1.43.0 upper-bound in setup.py here,
-        # see https://github.com/ray-project/ray/issues/28354
-        - grpcio >=1.32.0,<1.48
+        - grpcio >=1.32.0
         - jsonschema
         - msgpack-python >=1.0.0, <2.0.0
         - numpy >=1.20
         - pickle5  # [py<38]
```

Comment:
any `py<38` dependencies can be removed, right?

Reply:
See #81